test/edu-benefits/1990-rjsf/config/contributions.unit.spec.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-dom/test-utils';
+
+import { DefinitionTester, submitForm } from '../../../util/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/edu-benefits/1990-rjsf/config/form';
+
+describe('Edu 1990 contributions', () => {
+  const { schema, uiSchema } = formConfig.chapters.militaryHistory.pages.contributions;
+  const definitions = formConfig.defaultDefinitions;
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          data={{}}
+          uiSchema={uiSchema}
+          definitions={definitions}/>
+    );
+
+    const formDOM = findDOMNode(form);
+
+    expect(formDOM.querySelectorAll('input').length).to.equal(5);
+  });
+  it('should have no required inputs', () => {
+    const onSubmit = sinon.spy();
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          onSubmit={onSubmit}
+          data={{}}
+          uiSchema={uiSchema}
+          definitions={definitions}/>
+    );
+    const formDOM = findDOMNode(form);
+    submitForm(form);
+
+    expect(formDOM.querySelectorAll('.usa-input-error')).to.be.empty;
+    expect(onSubmit.called).to.be.true;
+  });
```

Comment:
I think this needs a test that clicks on the last contribution option and checks that the date fields are revealed.